ckanext/canada/helpers.py

```diff
@@ -774,6 +776,53 @@ def date_field(field, pkg):
     return pkg.get(field, None)
 
 
+def goc_auth():
```

Comment:
@RabiaSajjad You can get rid of `goc_auth`, and just put the:
```
remote_addr = request.headers.get('X-Forwarded-For') or \
                           request.environ.get('REMOTE_ADDR')
```
into the `__registry_network_access` method.

and rename that method to `registry_network_access` (no double underscore). and then it does not need to accept the `remote_addr` param.

You can then register the `registry_network_access` as a template helper in the `DataGCCAInternal` plugin class as we will need to use it in these template conditions:
- https://github.com/open-data/ckanext-canada/blob/master/ckanext/canada/templates/internal/error_document_template.html#L14
- https://github.com/open-data/ckanext-canada/blob/master/ckanext/canada/templates/internal/header.html#L29